lib/utils/getLintOptionOrConfigValue.mjs

```diff
@@ -0,0 +1,32 @@
+import getConfigForFile from '../getConfigForFile.mjs';
+
+/** @import {InternalApi, LinterOptions, Config} from 'stylelint' */
+
+/**
+ * Get a value of the specified lint option or configuration.
+ *
+ * @template T
+ * @param {Object} options
+ * @param {InternalApi} options.stylelint
+ * @param {keyof LinterOptions} options.option
+ * @param {keyof Config} options.config
+ * @param {T} [options.default]
+ * @returns {Promise<T>}
+ */
+export default async function getLintOptionOrConfigValue({
+	stylelint,
+	option,
+	config,
```

Comment:
[suggestion]
I noticed that in the current implementation, the `option` and `config` parameters always have the same name (e.g., `cache`, `allowEmptyInput`, `formatter`). 

Would it make sense to simplify the function signature to use a single parameter instead?
This could help prevent mistakes and make the API cleaner.

```suggestion
	name,
```

Reply:
Great, it's much simpler! Fixed via 78c37ad6d 👍🏼